### PR TITLE
Added Translations for Chinese Language

### PR DESCRIPTION
**Pull request by Varinder-KM**
Chinese Language was added.

### CODE REVIEW

File: src/main/res/values-zh/strings.xml [hunk 1]
```diff
@@ -0,0 +1,272 @@
...
+
+    <!-- Strings related to SignOut Preference-->
+    <string name="pref_title_sign_out">登出</string>
+    <string name="pref_title_account">帐户</string>
+    <string name="pref_sign_out_desc">从Amahi TV登出</string>
+    <string name="pref_action_not_defined">动作未定义</string>
+
+    <!--Strings related to Theme Preference -->
+    <string name="pref_title_light_theme">轻主题</string>
+    <string name="pref_key_light_theme" translatable="false">启用浅色主题</string>
```
Comment (prakharagarwal-dev) [inline src/main/res/values-zh/strings.xml]: @Varinder-KM Please remove the translations for the strings marked `translatable:false`

**Review by Prakhar-Agarwal-byte — COMMENTED** [review]
Please refer `Languages.md` on how to add translations. Here is the [link ](https://github.com/amahi/android/blob/beta/LANGUAGES.md)

File: src/main/res/values-zh/strings.xml (hunk 1 above)
Comment (Varinder-KM) [reply src/main/res/values-zh/strings.xml]: OK,I do

File: src/main/res/values-zh/strings.xml [hunk 2]
```diff
@@ -57,8 +57,6 @@
     <string name="message_no_files">文件下载出现在这里</string>
     <string name="message_create_and_try_again">Find files to download</string>
     <string name="message_connection_error">连接错误</string>
-    <string name="message_connect_and_try_again">请连接到互联网，然后重试。</string>
-    <string name="message_connect_to_server_and_try_again">请连接到服务器，然后重试。</string>
```
Comment (prakharagarwal-dev) [inline src/main/res/values-zh/strings.xml]: @Varinder-KM why did you remove these lines ?

File: src/main/res/values-zh/strings.xml [hunk 3]
```diff
@@ -103,12 +101,8 @@
     <string name="upload_file">上传文件</string>
 
     <!--ANDROID TV RELATED STRINGS-->
-    <string name="app_title" translatable="false">阿玛希电视</string>
     <string name="exit_title">出口</string>
     <string name="exit_message">你确定你要退出吗？</string>
-    <string name="sign_out_title">登出</string>
-    <string name="sign_out_message">您确定要退出吗？</string>
-    <string name="cancel">取消</string>
```
Comment (prakharagarwal-dev) [inline src/main/res/values-zh/strings.xml]: Same here

File: src/main/res/values-zh/strings.xml [hunk 4]
```diff
@@ -206,38 +187,17 @@
...
     <string name="pref_server_active_list_desc">选择要访问其共享的活动服务器。</string>
-    <string name="pref_server_select_key" translatable="false">选择服务器</string>
 
     <!-- Strings related to SignOut Preference-->
     <string name="pref_title_sign_out">登出</string>
     <string name="pref_title_account">帐户</string>
     <string name="pref_sign_out_desc">从Amahi TV登出</string>
     <string name="pref_action_not_defined">动作未定义</string>
-
-    <!--Strings related to Theme Preference -->
```
Comment (prakharagarwal-dev) [inline src/main/res/values-zh/strings.xml]: Same here

File: src/main/res/values-bn/strings.xml [hunk 5]
```diff
@@ -46,6 +46,8 @@
     <string name="language_te">তেলেগু</string>
     <string name="language_de">জার্মান</string>
     <string name="language_hi">হিন্দি</string>
+    <string name="language_zh">Chinese</string>
```
Comment (prakharagarwal-dev) [inline src/main/res/values-bn/strings.xml]: Please add correct translation in Bengali for the string "Chinese".

File: src/main/res/values-de/strings.xml [hunk 6]
```diff
@@ -182,7 +182,7 @@
     <string name="language_de">Deutsche</string>
     <string name="language_te">Telugu</string>
     <string name="language_hi">Hindi</string>
-
+    <string name="language_zh">Chinese</string>
```
Comment (prakharagarwal-dev) [inline src/main/res/values-de/strings.xml]: Please add correct translation in German for the string "Chinese".

File: src/main/res/values-es/strings.xml [hunk 7]
```diff
@@ -66,6 +66,8 @@
     <string name="language_te">Telugu</string>
     <string name="language_de">alemán</string>
     <string name="language_hi">Hindi</string>
+    <string name="language_zh">Chinese</string>
```
Comment (prakharagarwal-dev) [inline src/main/res/values-es/strings.xml]: Please add correct translation in Spanish for the string "Chinese".

File: src/main/res/values-fr/strings.xml [hunk 8]
```diff
@@ -164,6 +164,8 @@
     <string name="language_te">Telugu</string>
     <string name="language_de">Allemand</string>
     <string name="language_hi">Hindi</string>
+    <string name="language_zh">Chinese</string>
```
Comment (prakharagarwal-dev) [inline src/main/res/values-fr/strings.xml]: Please add correct translation in French for the string "Chinese".

File: src/main/res/values-hi/strings.xml [hunk 9]
```diff
@@ -133,6 +133,7 @@
     <string name="language_hi">हिंदी</string>
     <string name="language_te">तेलेगु</string>
     <string name="language_de">जर्मन</string>
+    <string name="language_zh">Chinese</string>
```
Comment (prakharagarwal-dev) [inline src/main/res/values-hi/strings.xml]: Please add correct translation in Hindi for the string "Chinese".

File: src/main/res/values-te/strings.xml [hunk 10]
```diff
@@ -48,6 +48,8 @@
     <string name="language_de">జర్మన్</string>
     <string name="language_te">తెలుగు</string>
     <string name="language_hi">హిందీ</string>
+    <string name="language_zh">Chinese</string>
```
Comment (prakharagarwal-dev) [inline src/main/res/values-te/strings.xml]: Please add correct translation in Telugu for the string "Chinese".

File: src/main/res/values/strings.xml [hunk 11]
```diff
@@ -200,7 +200,9 @@
     <string name="language_de">German</string>
     <string name="language_te">Telugu</string>
     <string name="language_hi">Hindi</string>
+    <string name="language_zh">Chinese</string>
 
+    <string name="language_zh_key" translatable="false">hi-rIN</string>
```
Comment (prakharagarwal-dev) [inline src/main/res/values/strings.xml]: change its value to "zh"

**Review by Prakhar-Agarwal-byte — CHANGES_REQUESTED** [review]

File: src/main/res/values-zh/strings.xml [hunk 12]
```diff
@@ -0,0 +1,232 @@
...
+    <string name="settings">设定值</string>
+    <string name="header_settings">设定值</string>
+    <string name="songs_in">内有歌曲 </string>
+    <string name="videos_in">中的视频 </string>
+
+    <string name="alert_delete_dialog">删除文件</string>
+    <string name="alert_delete_confirm">你确定吗？</string>
+
+    <string name="title_file_info">文件信息</string>
+    <string name="text_ok">OK</string>
```
Comment (prakharagarwal-dev) [inline src/main/res/values-zh/strings.xml]: Add Chinese translation for it.

File: src/main/res/values-zh/strings.xml [hunk 13]
```diff
@@ -0,0 +1,232 @@
...
+    <string name="message_upload_success">上传完成</string>
+    <string name="message_upload_error">上传失败</string>
+    <string name="message_file_download_complete">文件下载成功</string>
+    <string name="message_offline_file_deleted">离线文件删除成功</string>
+    <string name="notification_download_offline_title">使离线可用</string>
+    <string name="notification_offline_download_complete">文件可离线使用</string>
+    <string name="notification_offline_download_paused">等待上网</string>
+    <string name="notification_offline_download_error">使文件脱机可用时出错</string>
+
+    <string name="share_subject">Check out my Amahi home server!</string>
```
Comment (prakharagarwal-dev) [inline src/main/res/values-zh/strings.xml]: Add Chinese Translation for it.

File: src/main/res/values-zh/strings.xml [hunk 14]
```diff
@@ -0,0 +1,241 @@
...
+<resources>
+    <string name="title_apps">应用</string>
+    <string name="title_settings">设定值</string>
+    <string name="title_shares">分享</string>
+    <string name="title_no_server">没有伺服器</string>
+    <string name="title_upload_list">上载</string>
+    <string name="title_offline_files">离线文件</string>
+    <string name="title_recent_files">最近的文件</string>
+
+    <string name="button_search_application">搜索Google Play</string>
```
Comment (prakharagarwal-dev) [inline src/main/res/values-zh/strings.xml]: We have translated Google Play into other languages as well. Please translate it into Chinese also.

File: src/main/res/values-zh/strings.xml [hunk 15]
```diff
@@ -0,0 +1,241 @@
...
+    <string name="menu_save">保存到设备</string>
+    <string name="menu_delete">删除</string>
+    <string name="menu_offline">离线</string>
+    <string name="menu_sort">分类</string>
+    <string name="menu_search">搜索</string>
+    <string name="menu_open">打开</string>
+    <string name="menu_audio_files">档案</string>
+    <string name="menu_video_subtitle">字幕</string>
+
+    <string name="media_route_menu_item" translatable="false">投</string>
```
Comment (prakharagarwal-dev) [inline src/main/res/values-zh/strings.xml]: Please remove this line as it is marked `translatable="false"`

File: src/main/res/values-zh/strings.xml [hunk 16]
```diff
@@ -0,0 +1,241 @@
...
+    <string name="message_error_amahi_anywhere_app">服务器中的Amahi Anywhere应用程序无法访问。请确保您的网络中已安装，运行并可以访问Amahi Anywhere应用程序。</string>
+    <string name="message_resume_last_position">您要从停止的地方继续吗？</string>
+
+    <string name="message_notice_create_some">创建一些，然后重试</string>
+    <string name="message_notice_visit_some">造访一些，然后再试一次</string>
+
+    <string name="message_progress_file_downloading">下载文件</string>
+    <string name="message_progress_file_preparing">准备好的文件</string>
+    <string name="message_no_files">文件下载出现在这里</string>
+    <string name="message_create_and_try_again">Find files to download</string>
```
Comment (prakharagarwal-dev) [inline src/main/res/values-zh/strings.xml]: Translate this also

File: src/main/res/values-zh/strings.xml [hunk 17]
```diff
@@ -0,0 +1,241 @@
...
+    <string name="message_file_download_complete">文件下载成功</string>
+    <string name="message_offline_file_deleted">离线文件删除成功</string>
+    <string name="notification_download_offline_title">使离线可用</string>
+    <string name="notification_offline_download_complete">文件可离线使用</string>
+    <string name="notification_offline_download_paused">等待上网</string>
+    <string name="notification_offline_download_error">使文件脱机可用时出错</string>
+
+    <string name="share_subject">Check out my Amahi home server!</string>
+    <string name="share_message">我使用Amahi Home Server来存储，备份和流式传输所有文件。 \ n \ n看看！\ n \ n<a href="https://www.amahi.org/">https://www.amahi.org/</a></string>
+    <string name="title_version_settings">Amahi for Android</string>
```
Comment (prakharagarwal-dev) [inline src/main/res/values-zh/strings.xml]: Same here

File: src/main/res/values-zh/strings.xml [hunk 18]
```diff
@@ -0,0 +1,241 @@
...
+    <string name="exit_message">你确定你要退出吗？</string>
+    <string name="sign_out_title">登出</string>
+    <string name="sign_out_message">您确定要退出吗？</string>
+    <string name="cancel">取消</string>
+    <string name="yes">是</string>
+    <string name="no">没有</string>
+
+    <!--Introduction TV activity strings-->
+    <string name="intro_tv_1">欢迎使用Amahi进行Android TV</string>
+    <string name="intro_phone_1">欢迎使用Amahi for Android</string>
```
Comment (prakharagarwal-dev) [inline src/main/res/values-zh/strings.xml]: Please translate it properly including "for"

File: src/main/res/values-zh/strings.xml [hunk 19]
```diff
@@ -0,0 +1,241 @@
...
+
+    <!--    Languages-->
+    <string name="language_en">英语</string>
+    <string name="language_fr">法文</string>
+    <string name="language_es">西班牙文</string>
+    <string name="language_bn">孟加拉</string>
+    <string name="language_de">德语</string>
+    <string name="language_te">泰卢固语</string>
+    <string name="language_hi">印地语</string>
+    <string name="language_zh">Chinese</string>
```
Comment (prakharagarwal-dev) [inline src/main/res/values-zh/strings.xml]: Translate it into Chinese

File: src/main/res/values-zh/strings.xml [hunk 20]
```diff
@@ -0,0 +1,241 @@
...
+
+    <string name="preference_category_about">关于</string>
+    <string name="preference_category_account">帐户</string>
+    <string name="preference_category_settings">设定值</string>
+    <string name="preference_category_general">一般</string>
+    <String name="preference_key_language"></String>
```
Comment (prakharagarwal-dev) [inline src/main/res/values-zh/strings.xml]: Why is it empty?

**Review by Prakhar-Agarwal-byte — CHANGES_REQUESTED** [review]

File: src/main/res/values-zh/strings.xml [hunk 21]
```diff
@@ -0,0 +1,238 @@
...
+    <string name="message_delete_file_title">确认删除文件？</string>
+    <string name="message_delete_file_body">此操作无法撤消。</string>
+    <string name="message_delete_progress">删除</string>
+    <string name="message_delete_file_error">删除文件时发生一些错误。</string>
+    <string name="message_file_upload">选择要上传的文件</string>
+    <string name="message_file_upload_title">使用文件...</string>
+    <string name="message_file_upload_complete">文件上传成功</string>
+    <string name="message_file_upload_error">上传文件时出错。</string>
+    <string name="message_duplicate_file_upload">覆盖现有文件？</string>
+    <string name="message_duplicate_file_upload_body">文件％1 $ s已经存在。\ n确定要替换现有文件吗？</string>
```
Comment (prakharagarwal-dev) [inline src/main/res/values-zh/strings.xml:72]: Change "％1 $ s" to "%1$s" and "\ n" to "\n"

File: src/main/res/values-zh/strings.xml [hunk 22]
```diff
@@ -0,0 +1,238 @@
...
+    <string name="message_file_download_complete">文件下载成功</string>
+    <string name="message_offline_file_deleted">离线文件删除成功</string>
+    <string name="notification_download_offline_title">使离线可用</string>
+    <string name="notification_offline_download_complete">文件可离线使用</string>
+    <string name="notification_offline_download_paused">等待上网</string>
+    <string name="notification_offline_download_error">使文件脱机可用时出错</string>
+
+    <string name="share_subject">看看我的Amahi家用服务器！</string>
+    <string name="share_message">我使用 阿玛希 家庭服务器 来存储，备份和流式传输所有文件。 \ n \ n看看！\ n \ n<a href="https://www.amahi.org/">https://www.amahi.org/</a></string>
+    <string name="title_version_settings">阿玛希 对于 安卓</string>
```
Comment (prakharagarwal-dev) [inline src/main/res/values-zh/strings.xml]: The name Amahi should not be translated

File: src/main/res/values-zh/strings.xml [hunk 23]
```diff
@@ -0,0 +1,238 @@
...
+    <string name="exit_message">你确定你要退出吗？</string>
+    <string name="sign_out_title">登出</string>
+    <string name="sign_out_message">您确定要退出吗？</string>
+    <string name="cancel">取消</string>
+    <string name="yes">是</string>
+    <string name="no">没有</string>
+
+    <!--Introduction TV activity strings-->
+    <string name="intro_tv_1">欢迎使用Amahi进行 安卓 电视</string>
+    <string name="intro_phone_1">欢迎使用 阿玛希 对于 安卓</string>
```
Comment (prakharagarwal-dev) [inline src/main/res/values-zh/strings.xml]: same here

File: src/main/res/values/strings.xml [hunk 24]
```diff
@@ -200,7 +200,9 @@
     <string name="language_de">German</string>
     <string name="language_te">Telugu</string>
     <string name="language_hi">Hindi</string>
+    <string name="language_zh">Chinese</string>
 
+    <string name="language_zh_key" translatable="false">zh-rIN</string>
```
Comment (prakharagarwal-dev) [inline src/main/res/values/strings.xml]: Change "zh-rIN" to "zh".

**Review by Prakhar-Agarwal-byte — CHANGES_REQUESTED** [review]

File: src/main/res/values-zh/strings.xml (hunk 22 above)
Comment (prakharagarwal-dev) [reply src/main/res/values-zh/strings.xml]: Please recheck it in all translations

**Comment by Prakhar-Agarwal-byte** [comment]
Please complete the 3rd step in [Languages.md](https://github.com/amahi/android/blob/beta/LANGUAGES.md) file i.e add string item in string array in base strings.xml file

**Review by Prakhar-Agarwal-byte — APPROVED** [review]
Now it seems fine. Thank you for your contribution.

**Comment by Varinder-KM** [comment]
It's My Pleasure
